src/main/java/tinkerer/requests/github/prcomment/Comment.java

```diff
@@ -8,8 +8,10 @@
 @ToString
 @EqualsAndHashCode
 @NoArgsConstructor
-public class Repo {
-    //TODO: There are other repo objects in the payload. Might allow removal of head object. Check later.
+public class Comment {
     @JsonProperty
-    private String name;
+    private String url;
```

Comment:
Now future comments wont arrive either... this would require folks to comment on everything and then review at the close